superset/assets/src/explore/controlPanels/sections.jsx

```diff
@@ -91,6 +91,15 @@ export const subscribers = {
   ],
 };
 
+export const actions = {
+  label: t('Actions'),
+  expanded: true,
+  controlSetRows: [
+    ['navigate_to_dash_link_name'],
+    ['navigate_to_dashboards'],
+  ],
+};
+
```

Comment:
We should add a subsection "Navigate to Dashboard" and horizontal align this components as "Link Name" & "Dashboard"
Alos trim description as discussed

Reply:
done